package.json

```diff
@@ -1450,6 +1450,11 @@
           "default": 2097152,
           "description": "%mssql.query.maxXmlCharsToStore%"
         },
+        "mssql.autoDisableNonTSqlLanguageService": {
+          "type": "number",
```

Comment:
I think this makes more sense as a boolean with "undefined" permissible:

https://code.visualstudio.com/api/references/contribution-points#Configuration-example

```
        "typescript.tsdk": {
          "type": ["string", "null"],
          "default": null,
          "description": "Specifies the folder path containing the tsserver and lib*.d.ts files to use."
        }
```